en/exercises/voip_assessment/summary.md

```diff
@@ -0,0 +1 @@
+VoIP technologies are commonly used nowadays as it provides an alternate flexible way of communication. With its numerous benefits, from toll-bypass, unified voice and data trunking and universally accessible voice-mail and fax-mail services, VoIP services has indeed come into its place as one of the most used communication services today. However, with the rise of cyber attacks, and the reality that any device that connects online can be a potential risk for attacks, VoIP has been on of the favorite target of spam, Interruptions, Voice phishing Hacking and privacy loss.
```

Comment:
VOIP Services are commonly used nowadays as they provide a flexible alternative for communication. With numerous services and benefits like toll-bypass, unified voice and data trunking, global accessibility, voice-mail and fax mail services, VoIP is essential to the work of any organizations. On the other hand, VoIP has been one of the most targeted infrastructure by SPAM, Interruptions, hacking and privacy loss.